backend/pom.xml

```diff
@@ -182,6 +190,12 @@
       <artifactId>mssqlserver</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>net.java.dev.jna</groupId>
+      <artifactId>jna</artifactId>
+      <version>5.12.1</version>
```

Comment:
Can we define the version in the parent pom's `DependencyManagement` section like we do with the other dependencies?